bootstrap/bootstrap.py

```diff
@@ -8,7 +8,7 @@
     curl <script-url> | sudo python3 -
 
 Constraints:
-  - Entire script should be compatible with Python 3.6 (We run on Ubuntu 18.04+)
+  - Entire script should be compatible with Python 3.8 (We run on Ubuntu 20.04+)
   - Script should parse in Python 3.4 (since we exit with useful error message on Ubuntu 14.04+)
```

Comment:
```suggestion
  - Script should parse in Python 3.6 (since we exit with useful error message on Ubuntu 18.04+)
```
Since you've added f-strings.

More changes will be need though, e.g.
https://github.com/jupyterhub/the-littlest-jupyterhub/blob/8ba0ca2a9efbe1958c864cd338c6b79f025dff87/integration-tests/test_bootstrap.py#L87-L88

Reply:
Thanks @manics!